app/views/NextUpdates.scala.html

```diff
@@ -44,32 +49,29 @@ <h1 id="page-heading" class="heading-xlarge"> @messages("nextUpdates.heading")</
 		</summary>
 
 		<div class="govuk-details__text" id="details-content-0" data-module="govuk-details" aria-hidden="false">
-			<h2 id="quarterlyReturns-heading" class="heading-medium">@messages("obligations.quarterlyUpdates")</h2>
-			<p><span id="quarterly-dropdown-line1">@messages("nextUpdates.dropdown.quarterlyReturn.text")</span> </p>
-			<p><span id="quarterly-dropdown-line2">@Html(messages("nextUpdates.dropdown.quarterlyReturn.text.lin2",
-			))
-			</span>
-			</p>
+			<h2 id="quarterlyReturns-heading" class="govuk-heading-m">@messages("obligations.quarterlyUpdates")</h2>
+			@p(id = Some("quarterly-dropdown-line1"), content = Html(messages("nextUpdates.dropdown.quarterlyReturn.text")))
+			@p(id = Some("quarterly-dropdown-line2"), content = Html(messages("nextUpdates.dropdown.quarterlyReturn.text.lin2")))
```

Comment:
I don't see `<span` in the content here. Is it not needed?

Reply:
No <span> in the prototype so removed from the frontend, looks visually fine, and no accessibility errors from Wave